pyproject.toml

```diff
@@ -103,7 +103,7 @@ dependencies = [
     "python-snappy~=0.7.3",
     "python3-saml==1.16.0",
     "pytz==2023.3",
-    "redis==4.5.4",
+    "redis==6.4.0",
```

Comment:
that's a big jump
should we do it separately 🙈 

(not strongly held, i've not read the changelog for it :)))

Reply:
But I'll ask in dev anyway 👍

Reply:
much less scary as not the main redis 👍

Reply:
I would also vote to do it separately out of an abundance of caution.